packages/next/package.json

```diff
@@ -54,7 +57,7 @@
     "next-i18next": "^13.1.5",
     "react": "^18.0.2",
     "react-inlinesvg": "^3.0.1",
-    "xlsx": "^0.18.5"
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.19.3/xlsx-0.19.3.tgz"
```

Comment:
Just curious, what's the deal with this?

Reply:
xlsx isn't being published on npm because they have some legal issues (?), we had to do the same and bespoke and I couldn't find an easy replacement for the library, but I'd love to replace it in the future